crates/runtime/src/accelerated_table.rs

```diff
@@ -274,6 +274,10 @@ impl AcceleratedTable {
         Ok(())
     }
 
+    #[must_use] pub fn get_federated_table(&self) -> Arc<dyn TableProvider> {
```

Comment:
Should this return `self.federated`?

Reply:
Yes, sorry I didn't make a good comment. Essentially this was a draft hacky implementation (using spicepod details instead of just the `TableProvider` to ensure we had it this week). Solution needed this amongst other things.